.github/workflows/cherry-pick-to-frozen.yml

```
@@ -0,0 +1,276 @@
+          MAJOR="${VERSION%%.*}"
+          MINOR="${VERSION##*.}"
+          
+          # Compute next version
+          if [ "$MINOR" -eq 9 ]; then
+            NEXT_MAJOR=$((MAJOR + 1))
+            NEXT_MINOR=0
+          else
+            NEXT_MAJOR="$MAJOR"
+            NEXT_MINOR=$((MINOR + 1))
+          fi
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)